roles/dotfiles/templates/zprofile.j2

```diff
@@ -0,0 +1,39 @@
+{{ ansible_managed | comment }}
+
+export TERM="xterm-256color"
+
+# See https://github.com/Homebrew/brew/blob/master/Library/Homebrew/brew.sh#L48 for default value
+export HOMEBREW_CACHE={{ homebrew_cache_path }}
+
+# Load editor used by "dev config"
+[ -e "${HOME}/.dev_config_env" ] && source "${HOME}/.dev_config_env"
+
+# gpg and password prompt
+export GPG_TTY=$(tty)
+
+# need by vips
+export PKG_CONFIG_PATH="${PKG_CONFIG_PATH}:{{ brew_prefix }}/opt/libffi/lib/pkgconfig"
+
+# OpenSSL
+export PATH="{{ lookup('pipe', 'brew --prefix openssl') }}/bin:$PATH"
+
+# PHP
+[ -e "${HOME}/.load_php" ] && source "${HOME}/.load_php"
```

Comment:
This line must be loaded before the php-version-switcher plugin otherwise if we open a terminal in a folder with a .php-version file the default version takes precedence over the version indicated